src/CommunityToolkit.Maui.Core/Platform/iOS/PaddedButton.ios.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using UIKit;
+
+namespace Core.Platform.iOS;
+class PaddedButton : UIButton
```

Comment:
I think it shouldn't be in Core

Reply:
why?

Reply:
Because this is a control and it is not used in Core